package.json

```diff
@@ -4,9 +4,10 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "dev": "nodemon --exec babel-node ./src/index.js",
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
the coverage script does not run the tests, from which source is the report generated from?

Reply:
The issue was a source of the tests. Now, It is working.